pep-0612.rst

```diff
@@ -128,169 +156,346 @@ will defer the standardization of the semantics of those options to a later PEP.
 Valid use locations
 ^^^^^^^^^^^^^^^^^^^
 
-A declared ``ParameterSpecification`` can only be used in the place of the list
-of types in the declaration of a ``Callable`` type, or a user defined class
-which is generic in a ``ParameterSpecification`` variable (i.e., ``MyClass`` in
-the following example).
+Previously, the only things acceptable in the first "argument" to the
```

Comment:
nit: s/thing/entity or something like that?

Reply:
eh I chose to be purposely vague there since I didn't want to call it an expression etc.  
I think "only things" sounds reasonably idiomatic and achieves the goal of being imprecise.  If you think it's confusing then would be happy to change it, but if you just think its imprecise then I'd keep it.

Reply:
I think it's less about confusion or precision. To me the word "thing" just feels awkward in a professional documents where as "entity" is equally vague but feels better.

Reply:
I think I can actually restructure that sentence to avoid the need for a word their altogether